src/utils/output.ts

```diff
@@ -0,0 +1,3 @@
+export function writePlainOutput(...args: any[]) {
```

Comment:
I'm not sure this is the best name to explain that this function is the one to use to show output to a user. It also doesn't need to be used for "plain" output, as there are uses in this PR where `chalk` coloured text are sent to output.

Would it fit better as part of the logger class too? As discussed, all the log levels (debug/info/etc) pipe to stderr, and this function is the only one sending to stdout, but for other developers reaching to use the logger or the outputter, this might be a bit lost on its own. Might we consider `logger.output(...args)` as a method on the `logger` object that specifically will output things, whilst everything else is logged?

Or is that just complicating things more?

Reply:
I think this is complicating things more and the reason why I'm keeping the two separate is because that's how the Twilio CLI does it as well and if I'm playing around with this, I might as well create parity. 
The idea is that output is really only used for the "result". So if a command fails, the result should be empty for example.
The reason I introduced this to be more clear when to use output and when to log something. But I agree the naming might not be great. Maybe it's just `writeOutput`.

Reply:
`writeOutput` definitely works better for me than `writePlainOutput` 👍